cmd/scan.go

```diff
@@ -52,8 +54,9 @@ Scans a Go module directory. To scan the current directory recursively, use goka
 		globals, _ := cmd.Flags().GetBool("globalsTainted")
 		verbose, _ := cmd.Flags().GetBool("verbose")
 		debug, _ := cmd.Flags().GetBool("debug")
+		exitCode, _ := cmd.Flags().GetBool("exitCode")
 		util.InitConfig(globals, sarif, verbose, debug, outputPath, yml)
```

Comment:
I think we need to pass exitCode to initConfig as well to activate the flag

Reply:
Good point - we don't use it anywhere else at the moment, but at some point it will be nice to be able to check this agains `util.Config.ExitCode` from anywhere in the program.